lib/room.rb

```diff
@@ -1,12 +1,22 @@
-class Rom
+class Room
+  attr_reader :category
 
   def initialize(category, length, width)
     @category = category
-    @length = 10
-    @width = width
+    @length = length.to_i
```

Comment:
length is already an integer when it's passed in, so no particular need to convert to an integer here.